cylc/flow/job_file.py

```diff
@@ -63,6 +64,7 @@ def write(self, local_job_file_path, job_conf, check_syntax=True):
             with open(tmp_name, 'w') as handle:
                 self._write_header(handle, job_conf)
                 self._write_directives(handle, job_conf)
+                self._write_reinvokcation(self, handle):
```

Comment:
Spelling, and syntax error (colon!)
```suggestion
                self._write_reinvocation(self, handle)
```